### PR TITLE
[W-13315036] added js script for review site

### PR DESCRIPTION
**Pull request by agansop**

### CODE REVIEW

File: src/partials/head/head-scripts.hbs [hunk 1]
```diff
@@ -87,5 +87,17 @@ window.settings_timer=setTimeout(function () {_vwo_code.finish() },settings_tole
...
+    "openSidebar": true,
+    "openLoginForm": true
+  }
+</script>
+
+<script type="application/json" class="js-hypothesis-config">
+  {
+    "sidebarAppUrl": "https://dev-docs-review-h.kdev.msap.io/app.html"
+  }
+</script>
```
Comment (cheungaryk) [inline src/partials/head/head-scripts.hbs:101]: why have 2 different script blocks? Why not just:

``` javascript
<script type="application/json" class="js-hypothesis-config">
  {
    "openSidebar": true,
    "openLoginForm": true,
    "sidebarAppUrl": "https://dev-docs-review-h.kdev.msap.io/app.html"
  }
</script>
```

Comment (agansop) [reply src/partials/head/head-scripts.hbs:101]: It was provided by Mehdi I haven’t tested with only one block

Comment (agansop) [reply src/partials/head/head-scripts.hbs:101]: Let me test first with one block.

Comment (agansop) [reply src/partials/head/head-scripts.hbs:101]: Testing with one block does not work, h apps doesn’t open at all

**Review by cheungaryk — APPROVED** [review]

**Pull request merged**